modules/graceful/manager_unix.go

```diff
@@ -59,8 +59,8 @@ func (g *Manager) start() {
 	go func() {
 		defer func() {
 			close(startupDone)
-			// Close the unused listeners and ignore the error here there's not much we can do with it, they're logged in the CloseProvidedListeners function
-			_ = CloseProvidedListeners()
+			// Close the unused listeners
+			CloseProvidedListeners()
```

Comment:
If the function is invoked only by the package itself, maybe make it a private one.